pcsx2/gui/FrameForGS.cpp

```diff
@@ -438,7 +438,7 @@ GSFrame::~GSFrame() throw()
 void GSFrame::OnCloseWindow(wxCloseEvent& evt)
 {
 	sApp.OnGsFrameClosed( GetId() );
-	Hide();		// and don't close it.
+	CoreThread.Reset(); // reset thread to avoid suspend status on close event.
```

Comment:
> the problem is I couldn't find any such functions , so the only other logical alternative would be to do a reset on close event.

This sounds to me like "Oh, this guy has an infection at his finger, let's chop off his arm to make sure there's no more infection".